packages/matchbox/src/components/Modal/Modal.js

```diff
@@ -59,12 +65,22 @@ class Modal extends Component {
     );
 
     return (
-      <div className={modalClasses} {...rest} ref={(node) => this.container = node}>
+      <div className={modalClasses} onClose={onClose} {...rest} ref={(node) => this.container = node}>
         <Grid center='xs' middle='xs' className={styles.Grid}>
           <Grid.Column xs={11} md={9} xl={7}>
             <Content contentRef={(node) => this.content = node} open={open}>
               <WindowEvent event='keydown' handler={this.handleKeyDown} />
               <WindowEvent event='click' handler={this.handleOutsideClick} />
+              {showCloseButton && (
```

Comment:
If you put this outside <Content> I don't think you'll need the position absolute, maybe just a float right

Reply:
Also you wouldn't need to pass through onClose, since it'll be considered "outside"

Reply:
As discussed, I tried several different ways and this seemed to make the most sense.